src/TLuaInterpreter.cpp

```diff
@@ -10247,6 +10247,79 @@ int TLuaInterpreter::openWebPage(lua_State* L)
     return 1;
 }
 
```

Comment:
add
`// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setDiscordGame`
alike for the other functions

Reply:
Added